lib/services/manager_api.dart

```diff
@@ -129,9 +129,14 @@ class ManagerAPI {
     }
   }
 
+  Future<File> getKeyStoreFile() async {
```

Comment:
When this is merged, the previous Keystore file will be lost. Migration code is required.

Reply:
I have no idea to migrate the previous keystore. This code is just created from `initialize` of `patcher_api.dart` to make sure patcher, import, export and delete is using from the same file

Reply:
If we merge this code, will the previously saved keystore remain in tact?

Reply:
Yes

Reply:
How does it guarantee that? The previous location is `/sdcard/Android/data/app.revanced.manager.flutter/files/revanced-manager.keystore` but the new method `getKeyStoreFile()` uses `getTemporaryDirectory()` or `getExternalStorageDirectory()`. Does `getExternalStorageDirectory()` point to `/sdcard/Android/data/app.revanced.manager.flutter/files/`?

Reply:
1. `/sdcard/Android/data/app.revanced.manager.flutter/files/` only used in `deleteKeystore` method
2. the result of `getExternalStorageDirectory()` is `/storage/emulated/0/Android/data/app.revanced.manager.flutter/files` in my phone. it is same with `/sdcard/Android/data/app.revanced.manager.flutter/files`

Reply:
https://github.com/revanced/revanced-manager/tree/custom-keystore